packages/ts-moose-lib/src/streaming-functions/runner.ts

```diff
@@ -315,7 +319,9 @@ const sendMessages = async (
     let chunkSize = 0;
 
     for (const message of messages) {
-      const messageSize = Buffer.byteLength(message.value, "utf8");
+      const messageSize =
+        Buffer.byteLength(message.value, "utf8") +
```

Comment:
maybe a comment saying `message.key` is not set, so that does not take up any length.